pkg/network/tracer/offsetguess_test.go

```diff
@@ -291,6 +291,11 @@ func testOffsetGuess(t *testing.T) {
 			if kv >= kernel.VersionCode(5, 18, 0) {
 				continue
 			}
+		case offsetCtOrigin, offsetCtIno, offsetCtNetns, offsetCtReply:
```

Comment:
Aren't there other tests that need a similar fix? How about adding a small test to check if the kernel requirement works?

Reply:
the only tests I see failing in the CI tracker are `TestOffsetGuess` and `TestOffsetGuessPortIPv6Overlap`, which both use this helper `testOffsetGuess` to actually make the comparisons. 
I can add a test for the kernel requirement sure

Reply:
I meant with that the new requirement that we have. There should be other tests that are now going to fail with the new requirement.

Reply:
I see what you mean. I'll skip those as well for these kernels